src/odb/include/odb/db.h

```diff
@@ -7135,6 +7136,31 @@ class dbAccessPoint : public dbObject
   // User Code End dbAccessPoint
 };
 
+class dbBusPort : public dbObject
+{
+ public:
+  const char* getName() const;
+
+  int getStartIx() const;
+
+  int getSize() const;
+
+  bool isUpdown() const;
+
+  dbModule* getParent() const;
+
+  dbModBTerm* getFirstmember() const;
```

Comment:
Why is this public?  It would be better to have an iterator over the members

Reply:
It is public so the iterator can get the first member to start the iteration. Are you saying you would prefer to have it private and then add another duplicate method ? Or would you like to keep it private and make the iterator a friend of the class ?

Reply:
New iterators provided to hide the underlying structures.